modules/gitops-configuring-tls-for-redis-with-autotls-disabled.adoc

```diff
@@ -0,0 +1,222 @@
+// Module is included in the following assemblies:
+//
+// * /cicd/gitops/configuring-secure-communication-with-redis.adoc
+
+[id="gitops-configuring-tls-for-redis-with-autotls-disabled_{context}"]
+= Configuring TLS for Redis with autotls disabled
+
+.Procedure
+
+. Create an Argo CD instance:
+
+.. In the *Administrator* perspective of the web console, use the left navigation panel to go to *Administration* -> *CustomResourceDefinitions*.
+
+.. Search for `argocds.argoproj.io` and click `ArgoCD` custom resource definition (CRD).
+
+.. Click *Instances* -> *Create ArgoCD*.
+
+.. In the YAML field, replace the YAML code:
++
+.Example ArgoCD CR with autotls disabled
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: argocd <1>
+  namespace: openshift-gitops <2>
```

Comment:
@iam-veeramalla Is it okay to mention the namespace as **openshift-gitops** as the name of argocd nstance given here is **argocd** whereas we already have an instance **openshift-gitops** in that namespace?

Reply:
@varshab1210 I think that is ok, as in the description it is clearly mentioned what name and namespace represents.